static/app/components/events/interfaces/crashContent/exception/useSourceMapDebug.tsx

```diff
@@ -0,0 +1,125 @@
+import uniqBy from 'lodash/uniqBy';
+
+import type {ExceptionValue, Frame} from 'sentry/types';
+import {defined} from 'sentry/utils';
+import {QueryKey, useQuery, UseQueryOptions} from 'sentry/utils/queryClient';
+
+interface BaseSourceMapDebugError {
+  data: Record<string, string> | null;
```

Comment:
@scttcper won't this mean that we can basically do `data.anything` and not get a lint error? IMO I would just keep the base type as `data: null` since we shouldn't touch data except in cases where we have a specific error type.